static/app/components/feedback/feedbackSearch.tsx

```diff
@@ -64,6 +69,36 @@ function getSupportedTags(supportedTags: TagCollection) {
   };
 }
 
+/**
+ * Returns 2 sections: feedback fields and (filtered, non-field) tags.
+ */
+function getFilterKeySections(
+  supportedTags: TagCollection,
+  organization: Organization
+): FilterKeySection[] {
+  if (!organization.features.includes('search-query-builder-user-feedback')) {
+    return [];
+  }
+
+  const nonFieldTags: string[] = Object.values(supportedTags)
+    .map(tag => tag.key)
+    .filter(key => !FEEDBACK_FIELDS.includes(key as FeedbackFieldKey | FieldKey));
+  nonFieldTags.sort();
```

Comment:
Same comment as on the other PR, can sort these by `totalValues` to be a bit more useful